parsl/dataflow/dflow.py

```diff
@@ -10,7 +10,6 @@
 import threading
 import sys
 import datetime
-
```

Comment:
unnecessary whitespace change

Reply:
But, it was unnecessary to begin with.

Reply:
it's not to do with ad hoc cluster provision.